tests/x11/shutdown.pm

```diff
@@ -20,6 +20,13 @@ use utils;
 
 sub run() {
     my $self = shift;
+    x11_start_program('xterm');
+    script_sudo(q{echo 'ForwardToConsole=yes' >> /etc/systemd/journald.conf});
+    script_sudo(q{echo 'MaxLevelConsole=debug' >> /etc/systemd/journald.conf});
+    script_sudo(qq{echo 'TTYPath=/dev/$serialdev' >> /etc/systemd/journald.conf});
+    script_sudo(q{systemctl restart systemd-journald});
```

Comment:
I would actually propose to do this by default in consoletest_setup - the shutdown test isn't the only one benefiting from logs.